draft-ietf-quic-transport.md

```diff
@@ -2447,9 +2455,11 @@ Note that Stateless Reset packets do not have any cryptographic protection.
 
 ### Looping {#reset-looping}
 
-The design of a Stateless Reset is such that it is indistinguishable from a
-valid packet.  This means that a Stateless Reset might trigger the sending of a
-Stateless Reset in response, which could lead to infinite exchanges.
+The design of a Stateless Reset is such that without knowing the
+stateless reset token it is indistinguishable from a valid packet.
+This means that if a server sends a Stateless Reset to another server,
+that might trigger the sending of a Stateless Reset in response, which
```

Comment:
Can we get rid of one of these "that"s?

Reply:
feel free to suggest.

Reply:
"This means that if a server sends a Stateless Reset to another server, it might receive another Stateless Reset in response.  As a result...."

Reply:
One fewer: "If a server sends a Stateless Reset to another server, it might receive another Stateless Reset in response."